checker/src/org/checkerframework/checker/index/IndexChecker.java

```diff
@@ -13,7 +13,9 @@
  * <p>Constant Value Checker, SameLen Checker, SearchIndex Checker, Lower Bound Checker, Upper Bound
  * Checker
  *
- * <p>The Constant Value Checker has no dependencies.
+ * <p>The Constant Value Checker has no dependencies, but it does trust Positive annotations from
```

Comment:
That is the logical place for the documentation.  I think it's better to add documentation rather than to omit it because some other documentation is missing.